src/common/com/intellij/plugins/haxe/model/HaxeClassModel.java

```diff
@@ -72,6 +111,33 @@ public HaxeClassModel getParentClass() {
     return out;
   }
 
+  public HaxeClassModel getAliasOrSelf() {
+    final ResultHolder type = getAliasType();
+    final SpecificHaxeClassReference type1 = type != null ? type.getClassType() : null;
+    if (type1 != null) {
+      return type1.getHaxeClassModel();
+    }
+    return this;
+  }
+
+  @Nullable
+  public ResultHolder getAliasType() {
```

Comment:
The word "alias" as you're using it here means a typedef.  In Haxe documents, an alias is a pseudonym for a package name.  It would be better to use the term consistently with the Haxe documentation.